docs/tutorials/03_minimum_eigen_optimizer.ipynb

```diff
@@ -309,9 +309,9 @@
    "metadata": {},
    "source": [
     "### Analysis of Samples\n",
-    "`OptimizationResult` provides a useful information source `SolutionSample` (here denoted as *samples*). They contain\n",
-    "information about input values `x`, objective function values `fval`, probability of obtaining that result `probability`\n",
-    "and the solution status `status` (`SUCCESS`, `FAILURE`, `INFEASIBLE`)."
+    "`OptimizationResult` provides useful information in the form of `SolutionSample`s (here denoted as *samples*). They contain\n",
+    "information about input values `x`, objective function values (`fval`), the probability for having obtained the given result (`probability`),\n",
```

Comment:
I see you put  () around the field names the text is referring to, like `fval` and `probability`. Should that not then be done with `x` for consistency. I must admit I find `probability of that that result` which perhaps could be `probability of obtaining this result` or `...to obtain...` a lot easier to parse than `...for having obtained the given...`

Reply:
I left parentheses around identifiers that are parenthetical to the sentence and removed them for words that made sense as part of the sentence.

My goal with this wording was to clarify that this is the empirical probability that something _happened_, rather than a theoretical probability of whether something _might_ happen.

Reply:
Ok, it did feel like I was reading legalese when parsing it! I see the intent. I still find it tougher reading it though!

Reply:
I wonder if there's an even better way to express it.  I'll try to think of alternatives, and am open to suggestions. :slightly_smiling_face:

Reply:
@woodsp-ibm What do you think of my clarification in a37f93e038bd17ebe59a18f226a9b2d74fab109e?

Reply:
Any comment @woodsp-ibm?

Reply:
@t-imamichi Sorry, it seemed fine to me now in terms of consistency around the markup. I had seen you had approved it and figured it would soon be merged hence I did not comment any further at the time.

Reply:
OK. Thanks!